res/themes/light/css/_light.pcss

```diff
@@ -201,6 +201,8 @@ $e2e-verified-color: #76cfa5; /* N.B. *NOT* the same as $accent */
 $e2e-unknown-color: #e8bf37;
 $e2e-unverified-color: #e8bf37;
 $e2e-warning-color: #ba6363;
+$e2e-verified-color-light: #f1faf9;
+$e2e-warning-color-light: #fff5f5;
```

Comment:
Is this based on one of the compound colors?

Reply:
These are based on new designs from Jano on figma, which I assume are based on compound

Reply:
Then we should definitely be calling these what they are called in Compoud. And we should also look for any vars that are using the same values